.travis.yml

```diff
@@ -216,7 +216,7 @@ addons:
       - libpcap-dev
       - xsltproc
       - libxml2-utils
-      - libsystemd-journal-dev
+      - libsystemd-dev
```

Comment:
If it is possible to override `dist` in a job, maybe we could still run the journalbeat tests with older versions of ubuntu to avoid breaking compatibility with older systemd versions, @kvch wdyt?

In any case reconsidering this could go in a follow-up, let's fix `make check` first :slightly_smiling_face:

Reply:
I'd be fine with investigating setting `dist` specifically for journalbeat in a follow-up.